### PR TITLE
Add support for component-version messages.

### PR DESCRIPTION
**Pull request by ralphbean**
This is for FACTORY-2947.

We'll also have to add this to the list of topics that resultsdb-updater is
configured to listen for in production.

### CODE REVIEW

File: tests/fake_messages/pelc_component_version.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            "namespace":"pelc",
+            "run":{
+                "log":"https://rcm-tools-jenkins/3/console",
+                "url":"https://rcm-tools-jenkins/3/"
+            },
+            "status":"PASSED",
+            "type":"scan",
+            "version":"0.1.0"
+        },
+        "msg_id":"ID:rcm-tools-jenkins.rhev-ci-vms.eng.rdu2.redhat.com-33832-1533327968729-94211:1:1:1:1",
```
Comment (hluk) [inline tests/fake_messages/pelc_component_version.json]: Obfuscate the URL.

**Comment by ralphbean** [comment]
Just rebased on top of #68.

**Comment by ralphbean** [comment]
@gnaponie, can you review?

File: tests/fake_messages/pelc_component_version.json (hunk 1 above)
Comment (ralphbean) [reply tests/fake_messages/pelc_component_version.json]: Done.

**Comment by gnaponie** [comment]
Late +1. Thank you Ralph.

**Pull request merged**